ethereum/tokemak/view_tokemak_convex_pool_stats_daily.sql

```diff
@@ -0,0 +1,154 @@
+CREATE MATERIALIZED VIEW tokemak.view_tokemak_convex_pool_stats_daily
+(   
+    source
+    ,"date"
+    ,pool_address
+    ,pool_symbol
+    ,token_address
+    ,token_symbol
+    ,total_lp_supply
+    ,reserve
+) AS (
+    
+WITH  convex_pools As 
+    (
+    
+      --SELECT contract_address as pool_address, symbol as pool_symbol, sum(qty) OVER (PARTITION BY symbol ORDER BY "date") as pool_lp_qty FROM (
+        SELECT contract_address as pool_address, symbol, sum(qty) as qty 
+            FROM (
+            SELECT contract_address,tl.symbol, (value/10^tl.decimals)*-1 as qty 
+            FROM erc20."ERC20_evt_Transfer" t
+            INNER JOIN dune_user_generated.tokemak_lookup_tokens tl on tl.address = t.contract_address
```

Comment:
hi @mfuribondo -- please refrain from referring to 'dune_user_generated' tables in the repo code. if you need data from these tables, it's best to recreate the user generated table as a view/table in the repo first, then revisit these views. this comment applies to any other script that fits the same scenario 👍

Reply:
got it, in that case i will be checking in a couple more .sql files.  will notify you when ready to rereview , thanks!